auth-jwt-service/src/test/java/io/stargate/auth/jwt/AuthJWTServiceActivatorTest.java

```diff
@@ -41,6 +41,7 @@ public void shouldRegisterIfSelectedAuthProvider() {
 
     activator.start(bundleContext);
 
+    @SuppressWarnings("JdkObsolete")
```

Comment:
Why not replace by a `HashMap`? It doesn't seem we need thread safety.

Reply:
The OSGi method where this is used declares `Dictionary`, `HashMap` doesn't implement it.

Reply:
I wonder if using `Dictionary` as the type of parameters is going to please ErrorProne :thinking: